test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/FhirStorageTests.cs

```diff
@@ -163,14 +163,14 @@ private async Task UpdateResource(Patient patient)
         }
 
         [Fact]
-        public async Task GivenAResource_WhenSaving_ThenTheMetaIsUpdated_AndLastUpdatedIsWithin100ms()
+        public async Task GivenAResource_WhenSaving_ThenTheMetaIsUpdated_AndLastUpdatedIsWithin1sec()
         {
             var saveResult = await Mediator.UpsertResourceAsync(Samples.GetJsonSample("Weight"));
             Assert.NotNull(saveResult);
             Assert.Equal(SaveOutcomeType.Created, saveResult.Outcome);
             var deserializedResource = saveResult.RawResourceElement.ToResourceElement(Deserializers.ResourceDeserializer);
             Assert.NotNull(deserializedResource);
-            Assert.True((DateTimeOffset.UtcNow - deserializedResource.LastUpdated.GetValueOrDefault()).TotalMilliseconds < 100);
+            Assert.True((DateTimeOffset.UtcNow - deserializedResource.LastUpdated.GetValueOrDefault()).TotalMilliseconds < 1000);
```

Comment:
Suggestion: a more resilient fix might be a range check. I.e. checking that the value of `deserializedResource.LastUpdated` is between "now" captured as a variable before ln168 and "now" captured on ln169.

Reply:
That is a good suggestion. I will put it in my next PR.